app/views/equality_intro.html

```diff
@@ -33,7 +33,7 @@ <h1 class="govuk-heading-xl">We have received your application</h1>
     Do you want to answer the equality questions?
   </legend>
   <div id="equalities-info-hint" class="govuk-hint">
-    <p class="govuk-!-margin-top-0">These questions are optional. [Add a couple of sentences explaining why you’re asking the questions and what you’ll do with the information].</p>
+    <p class="govuk-!-margin-top-0">These questions are optional. These questions are optional. Your responses will help us to monitor and ensure that legal aid is being delivered fairly and that we do not discriminate in our decision making. The data will only be used anonymously and your individual responses will never be shared outside the legal aid agency.</p>
```

Comment:
'These questions are optional' is repeated, is that intentional?

Reply:
That is not intentional. I have fixed that now @oliverweighell

Reply:
Thank you for spotting it!